datadog/datachecks.q

```diff
@@ -0,0 +1,51 @@
+//DATADOG CHECKS
+//Script for running checks manually/via crontab
+//Makes dictionary of defaults and uses .Q.opt to refer to command values passed by its key.
+o:.Q.def[`user`pass`timeout`init`noexit!(`admin;`admin;100;1b;0b);.Q.opt[.z.x]]
+
+ddconfigfile:hsym `$getenv[`KDBAPPCONFIG],"/ddconfig.txt"
+
+//sets dogstatsd_port to the port defined by ddconfigfile
+$[`ddconfig.txt in key hsym `$getenv[`KDBAPPCONFIG];value each read0 ddconfigfile;dogstatsd_port:8125]
+
+// Send To Datadog function - takes a non string value and a stringed name.
+//Will send the value received (1 or 0) and the process name (hdb etc)
+.datadog.sendMetric:{[metric_name;metric_value] system"bash -c \"echo  -n '",metric_name,":",(string metric_value),"|g|#shell' > /dev/udp/127.0.0.1/",(string dogstatsd_port),"\"";}
+
+.datadog.sendEvent:{[event_title;event_text;tags;alert_type] system"event_title=",event_title,"; event_text=","\"",event_text,"\"","; tags=","\"",$[0h=type tags;"#",("," sv tags);"#",tags],"\"",";alert_type=",alert_type,"; ","echo \"_e{${#event_title},${#event_text}}:$event_title|$event_text|#$tags|t:$alert_type\" |nc -4u -w0 127.0.0.1 ",string dogstatsd_port;}
```

Comment:
Is this a duplicated? And if so why? Can we define these functions in one place only?